File: CoreJava/account-polymorphism/src/account/polymorphism/SavingAccount.java
package account.polymorphism;

public class SavingAccount extends Account {

	private static final double MINIMUM_BALANCE = 500.0;

	public SavingAccount(String n, double b) {
		super(n, b);
	}

	@Override
	public void withdraw(double amt) {
		countWithdraw+=1;
		if (balance - amt < MINIMUM_BALANCE) {
			System.out.println("Insufficient Balance");
		} else
			balance -= amt;

	}

}

File: C#/OOP/PolymorphismExample/PolymorphismExample/SavingAccount.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace PolymorphismExample
{
    class SavingAccount : Account
    {
        private static readonly double MINIMUM_BALANCE = 500.0;

        public SavingAccount(String n, double b)
            : base(n, b)
        {
        }


        public override void withdraw(double amt)
        {
            countWithdraw += 1;
            if (balance - amt < MINIMUM_BALANCE)
            {
                Console.WriteLine("Insufficient Balance");
            }
            else
                balance -= amt;

        }
    }
}
